--- ./config/prefs.php.dist	2004-04-15 05:09:54.000000000 +0300
+++ ./config/prefs.php	2005-02-27 21:56:46.000000000 +0200
@@ -6,7 +6,7 @@
  */
 
 // Make sure that INGO_STORAGE_* constants are defined.
-require_once dirname(__FILE__) . '/../lib/Storage.php';
+require_once '/usr/share/horde/ingo/lib/Storage.php';
 
 // This preference group will only be displayed if the configured
 // Ingo_Script:: driver can create script files.
